plugins/services/src/js/components/forms/MultiContainerNetworkingFormSection.js

```diff
@@ -160,15 +168,36 @@ class MultiContainerNetworkingFormSection extends mixin(StoreMixin) {
       address = `${hostname}${Networking.L4LB_ADDRESS}${port}`;
     }
 
+    const loadBalancerTooltipContent = (
+      <span>
+        {`Load balance the service internally (layer 4), and create a service
+        address. For external (layer 7) load balancing, create an external
+        load balancer and attach this service. `}
+        <a href="https://docs.mesosphere.com/usage/service-discovery/load-balancing-vips/"
```

Comment:
We do have a `MetadataStore.buildDocsURI` function to build doc links with the correct version!